aspnet/mvc/views/partial/sample/global.json

```diff
@@ -0,0 +1,6 @@
+﻿{
+  "projects": [ "src", "test" ],
+  "sdk": {
+    "version": "1.0.0-preview1-002702"
+  }
+}
```

Comment:
@danroth27  Can we add all these files to .ignore ? Can't you override .ignore when you want to include a default ignored file?

I see your point - that's very inconvenient. My project lives outside the Git tree and I have a robocopy job run every minute to copy over the bits I want. I'm not suggesting you use that work flow. We really need to come up with a plan to minimizing files checked in.